modules/metrics/mule-metrics-exporter-impl/src/main/java/org/mule/runtime/metrics/exporter/impl/OpenTelemetryMeterExporter.java

```diff
@@ -6,58 +6,109 @@
  */
 package org.mule.runtime.metrics.exporter.impl;
 
+import static org.mule.runtime.metrics.exporter.api.MeterExporterProperties.METRIC_EXPORTER_ENDPOINT;
+
+import static java.lang.System.getProperty;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+import io.opentelemetry.api.metrics.LongCounterBuilder;
```

Comment:
Import order (review all imports since there more than one that needs re arrange)